### PR TITLE
feat(dedicated-ip): rework alerts mechanism

### PR DESCRIPTION
**Pull request by y4nnL**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/ip-strat`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-9808
| License          | BSD 3-Clause

- [X] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [X] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/ip/components/alerts/alerts.controller.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      return 'error';
+    }
+    if (this.alerts && !this.countAlerts()) {
+      return 'success';
+    }
+    return 'warning';
+  }
+
+  get serviceTypeSuffix() {
+    return `_w${this.serviceType ? '' : 'o'}_service_type`;
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/alerts/alerts.controller.js:21]: **question:** What does `w` and `o` suggest here? Can we avoid abbreviations and use full name instead?

File: packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -33,18 +33,18 @@
   "ip_alerts_service_other": "{} alertes pour ce service",
   "ip_alerts_spam": "Aucune IP bloquée (Antispam)",
   "ip_alerts_spam_1": "1 IP bloquée (Antispam)",
-  "ip_alerts_spam_help": "Spam (port 25 bloqué suite à une détection d’activité de spam sur l’adresse IP)",
+  "ip_alerts_spam_help": "<strong>Spam</strong> (port 25 bloqué suite à une détection d’activité de spam sur l’adresse IP)",
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json:36]: **question:** Can we split into 2 sentences and avoid HTML In translation files.

Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json:36]: **question:** Can we split into 2 sentences and avoid HTML In translation files.

File: packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -33,18 +33,18 @@
...
-  "ip_alerts_antihack_help": "Anti-hack (blocage du service suite à une suspicion d’activité malveillante)",
+  "ip_alerts_antihack_help": "<strong>Anti-hack</strong> (blocage du service suite à une suspicion d’activité malveillante)",
   "ip_alerts_antihack_other": "{} IPs bloquées (Anti-hack)",
   "ip_alerts_arp": "Aucune IP bloquée (ARP)",
   "ip_alerts_arp_1": "1 IP bloquée (ARP)",
   "ip_alerts_arp_other": "{} IPs bloquées (ARP)",
   "ip_alerts_mitigation": "Aucune IP attaquée (Mitigation)",
   "ip_alerts_mitigation_1": "1 IP attaquée (Mitigation)",
-  "ip_alerts_mitigation_help": "Mitigation d’attaque DDoS (protection du service subissant une attaque DDoS)",
+  "ip_alerts_mitigation_help": "<strong>Mitigation d’attaque DDoS</strong> (protection du service subissant une attaque DDoS)",
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json:47]: **question:** Can we split into 2 sentences and avoid HTML In translation files.

File: packages/manager/apps/dedicated/client/app/ip/components/alerts/alerts.controller.js (hunk 1 above)
Comment (y4nnL) [reply packages/manager/apps/dedicated/client/app/ip/components/alerts/alerts.controller.js]: Actually this is more like "w" and "w/o" that stands for "with" and "without"
Fixed :+1:

**Review by anooparveti — APPROVED** [review]

**Review by vks-ovh — APPROVED** [review]

**Pull request merged**